columnflow/tasks/cutflow.py

```diff
@@ -190,7 +200,7 @@ def get_plot_parameters(self):
         params = super().get_plot_parameters()
 
         # no ratio plot implemented: disable ratio plot for `plot_cutflow`
-        if self.plot_function_name == "columnflow.plotting.example.plot_cutflow":
+        if self.plot_function_name_1d == "columnflow.plotting.example.plot_cutflow":
```

Comment:
Maybe `skip_ratio` should rather become a parameter? Having plot function dependent settings somehow works around the function plug-in mechanism and might not scale too well.

Reply:
`skip_ratio` already is a parameter, I just set it to True here, because this plot function does not provide a ratio plot. But I think, the ratio plot would not be produced event when `skip_ratio=False`, so I could also just remove this line.

Reply:
👍

Reply:
turns out I can't just remove this because then an empty ratio plot is drawn, but I simply set `skip_ratio=True` in the plot_function itself